### PR TITLE
FIX: docstring example in cohen_kappa.py

### PR DESCRIPTION
**Pull request by keshan**
Closes https://github.com/tensorflow/addons/issues/397
Corrected the example in docstring.

### CODE REVIEW

**Comment by WindQAQ** [comment]
cc @SSaishruthi

**Comment by SSaishruthi** [comment]
Thanks, @keshan for the PR.
With the specified weights, I am getting the result to be `0.37209308`. Can you please update the result?

Also, can we add an example for `sample_weight = None`?

**Comment by keshan** [comment]
> Thanks, @keshan for the PR.
> With the specified weights, I am getting the result to be `0.37209308`. Can you please update the result?
> 
> Also, can we add an example for `sample_weight = None`?

@SSaishruthi thanks for the feedback added the changes. PTAL. Thanks!

File: tensorflow_addons/metrics/cohens_kappa.py [hunk 1]
```diff
@@ -41,10 +41,16 @@ class CohenKappa(Metric):
     ```python
     actuals = np.array([4, 4, 3, 4, 2, 4, 1, 1], dtype=np.int32)
     preds = np.array([4, 4, 3, 4, 4, 2, 1, 1], dtype=np.int32)
-
-    m = tf.keras.metrics.CohenKappa(num_classes=5)
-    m.update_state(actuals, preds, "quadratic")
-    print('Final result: ', m.result().numpy()) # Result: 0.68932
+    weights = np.array([1, 1, 2, 5, 10, 2, 3, 3], dtype=np.int32)
+    
+    m = tensorflow_addons.metrics.CohenKappa(num_classes=5)
```
Comment (WindQAQ) [inline tensorflow_addons/metrics/cohens_kappa.py]: maybe change to `tfa.metrics.CohenKappa`?

Comment (WindQAQ) [inline tensorflow_addons/metrics/cohens_kappa.py]: maybe change to `tfa.metrics.CohenKappa`?

File: tensorflow_addons/metrics/cohens_kappa.py [hunk 2]
```diff
@@ -41,10 +41,16 @@ class CohenKappa(Metric):
...
+    m.update_state(actuals, preds)
+    print('Final result: ', m.result().numpy()) # Result: 0.61904764
+    
+    # To use this with weights, sample_weight argument can be used.
+    m = tensorflow_addons.metrics.CohenKappa(num_classes=5)
```
Comment (WindQAQ) [inline tensorflow_addons/metrics/cohens_kappa.py]: ditto

File: tensorflow_addons/metrics/cohens_kappa.py (hunk 1 above)
Comment (keshan) [reply tensorflow_addons/metrics/cohens_kappa.py]: Done.

File: tensorflow_addons/metrics/cohens_kappa.py (hunk 2 above)
Comment (keshan) [reply tensorflow_addons/metrics/cohens_kappa.py]: Done

**Comment by keshan** [comment]
@WindQAQ Done. PTAL. Thanks!
